src/pages/home/home.ts

```diff
@@ -26,12 +26,16 @@ export class HomePage {
           break;
         }
         case 'Edit': {
-          console.log('Edit');
+          this.navCtrl.push(ItemPage, {
```

Comment:
Good catch. Turns out we cannot use strings in TypeScript's enums, so I used a static class instead.